extensions/open-in-visual-studio-code/src/index.ts

```diff
@@ -32,7 +32,9 @@ const getSelectedFinderWindow = (): Promise<string> => {
 
 export default async () => {
   const applications = await getApplications();
-  const visualStudioCode = applications.find((app) => app.bundleId === "com.microsoft.VSCode");
+  const visualStudioCode = applications.find(
+    (app) => app.bundleId === "com.microsoft.VSCode" || app.bundleId === "com.vscodium"
+  );
```

Comment:
This logic can have different results, depending on the order of the applications. Also, what happens to people that have both versions installed? 

I would suggest to do the following:
1. Search for VS Code and VS Codium separately
2. Combine the results with an optional ternary operator. 

Alternatively, you can order the applications by bundle identifier and then use `find` as you do. An improvement could be to add a preference to the extension as a dropdown to select which version of VS Code should be used. By default, you could have an "automatic" way that prefers VS Code over Codium.